### ISSUE TITLE
Unrelated contracts get deployed when using create

### ISSUE THREAD

**Issue by nventuro** [OpenZeppelin/openzeppelin-sdk#1250]
I have a simple project with two unrelated contracts: `Receiver` and `Empty`. After running `oz create` and choosing to deploy `Receiver`, `Empty` was also deployed:

```bash
$ npx oz create 
Nothing to compile, all contracts are up to date.
? Pick a contract to instantiate Receiver
? Pick a network ropsten
✓ Contract Receiver deployed
✓ Contract Empty deployed
All contracts have been deployed
```

I understand this may be related to an underlying `push` operation, but it is an unexpected outcome which may result in the user's funds being used in ways they didn't intend or agree to (there was no prompt asking whether or not I wanted to deploy the other contract on my project).

**Comment by gitpusha** [comment]
I want to raise the same issue. However, in my case I checked whether the unwanted deployment actually took place, and I found out that it did not take place. Still, the console output was misleading.

**Comment by nventuro** [comment]
@gitpusha when using `oz create` to deploy upgradeable contracts, first an implementation has to be deployed, and then a proxy pointing to it (the instance you actually use). I think the issue here is that `oz create` deploys implementations for all contracts in the directory, regardless of them being needed by your command or not, and logs that as 'Contract X deployed'. 

Now that `oz create` is more 'magical' and hides this underlying complexity, we should at the very least hide or change the wording of these messages.

**Comment by gitpusha** [comment]
@nventuro ok, understood. It would be super sweet if `oz create` would be isolated to the specified `contract name`/`alias` I supply to the command. It is a little troublesome when you are coding and just want to create and test single contract instances on the `development` (`ganache`) network. At the moment I cannot test a single `contracts-ethereum-package` out because I cannot create it on the development network by itself before fixing all other contracts in my project first. Apparently one of them cannot be `oz created` and thus is blocking the `oz creation` of the contract that I am actually currently working on  - see:
![image](https://user-images.githubusercontent.com/36712489/67617973-04b62f80-f7e9-11e9-9022-fcc7f6edf27c.png)

**Comment by abcoathup** [comment]
I tried deploying a contract with another contract in the same project and only the contract I deployed was created (along with the Proxy and ProxyAdmin contracts) as I would expect (in both development and Ropsten).

## A.sol
```
pragma solidity ^0.5.0;

contract A {
}
```
```
## B.sol
```
pragma solidity ^0.5.0;

contract B {
}
```
$ oz create
✓ Compiled contracts with solc 0.5.12 (commit.7709ece9)
? Pick a contract to instantiate A
? Pick a network development
✓ Added contract A
✓ Contract A deployed
All contracts have been deployed
✓ Setting everything up to create contract instances
✓ Instance created at 0x2c5f3c004878923f55A2a255F89Fe29393177509
0x2c5f3c004878923f55A2a255F89Fe29393177509
```

**Comment by nventuro** [comment]
@abcoathup that may be a bug, I tried the same example you pasted, and while using `create` on A only deploys A, running it on B deploys both A and B.

**Comment by abcoathup** [comment]
@nventuro 
My manual testing also shows contracts in `project.json` `contracts` section get deployed regardless of what is called in `oz create`.

**Comment by gitpusha** [comment]
Was this issue resolved in the recent `oz version 2.6.0` release?
